src/test/java/com/monumental/services/MonumentServiceTests.java

```diff
@@ -0,0 +1,175 @@
+
+        when(this.session.createQuery("FROM com.monumental.models.Monument")).thenReturn(this.getAllQuery);
+
+        // Setup Query mock
+        when(this.getAllQuery.list()).thenReturn(this.monuments);
+    }
+
+    /**
+     * Test method for unit testing MonumentService.insert(record)
+     * Mocks the appropriate classes as to not connect to the database
+     */
+    @Test
+    public void unitTestMonumentServiceInsert_Single() {
+        int result = this.monumentService.insert(this.monument1);
+
+        assertEquals(1, result);
+    }
+
+    /**
+     * Test method for unit testing MonumentService.insert(List<record>)
+     * Mocks the appropriate classes as to not connect to the database
+     */
+    @Test
+    public void unitTestMonumentServiceInsert_Multiple() {
+        List<Integer> results = this.monumentService.insert(this.monuments);
+
+        assertEquals(3, results.size());
+
+        Integer result1 = results.get(0);
+        Integer result2 = results.get(1);
+        Integer result3 = results.get(2);
+
+        assertEquals((Integer) 1, result1);
+        assertEquals((Integer) 2, result2);
+        assertEquals((Integer) 3, result3);
+    }
+
+    /**
+     * Test method for unit testing MonumentService.get(record)
+     * Mocks the appropriate classes as to not connect to the database
+     */
+    @Test
+    public void unitTestMonumentServiceGet_Single() {
+        Monument result = this.monumentService.get(1);
+
+        assertEquals(this.monument1.getTitle(), result.getTitle());
+    }
+
+    /**
+     * Test method for unit testing MonumentService.getAll(List<record>)
+     * Mocks the appropriate classes as to not connect to the database
+     */
+    @Test
+    public void unitTestMonumentServiceGetAll_ListPassed() {
+        List<Monument> results = this.monumentService.getAll(this.ids);
+
+        assertEquals(3, results.size());
+
+        Monument result1 = results.get(0);
+        Monument result2 = results.get(1);
+        Monument result3 = results.get(2);
+
+        assertEquals(this.monument1.getTitle(), result1.getTitle());
+        assertEquals(this.monument2.getTitle(), result2.getTitle());
+        assertEquals(this.monument3.getTitle(), result3.getTitle());
+    }
+
+    /**
+     * Test method for unit testing MonumentService.getAll(null)
+     * Mocks the appropriate classes as to not connect to the database
+     */
+    @Test
+    public void unitTestMonumentServiceGetAll_NullPassed() {
+        List<Monument> results = this.monumentService.getAll();
+
+        assertEquals(3, results.size());
+
+        Monument result1 = results.get(0);
+        Monument result2 = results.get(1);
+        Monument result3 = results.get(2);
+
+        assertEquals(this.monument1.getTitle(), result1.getTitle());
+        assertEquals(this.monument2.getTitle(), result2.getTitle());
+        assertEquals(this.monument3.getTitle(), result3.getTitle());
+    }
+}
```

Comment:
I didn't add any unit tests for `MonumentService.update()` or `MonumentService.delete()` because they do not have return values, so there's nothing to assert on. It would basically be testing that the mock does what I tell it to do. These will be covered in the integration tests.